external/googleapis/protolists/discoveryengine.list

```diff
@@ -1,20 +1,30 @@
+@com_google_googleapis//google/cloud/discoveryengine/v1:answer.proto
 @com_google_googleapis//google/cloud/discoveryengine/v1:common.proto
 @com_google_googleapis//google/cloud/discoveryengine/v1:completion.proto
 @com_google_googleapis//google/cloud/discoveryengine/v1:completion_service.proto
+@com_google_googleapis//google/cloud/discoveryengine/v1:control.proto
+@com_google_googleapis//google/cloud/discoveryengine/v1:control_service.proto
```

Comment:
Many of these `*_service.proto` files look like they contain new services. If there are no existing feature requests, consider opening one to add these files to `generator_config.textproto`.